web/app.js

```diff
@@ -1739,14 +1741,41 @@ function webViewerPageRendered(evt) {
     });
     // It is a good time to report stream and font types.
     PDFViewerApplication.pdfDocument.getStats().then(function (stats) {
+      // Convert the stats to telemetry friendly names.
+      let newStats = {
+        streamTypes: toTelemtryNames(swapKeyValue(StreamType),
+                                     stats.streamTypes),
+        fontTypes: toTelemtryNames(swapKeyValue(FontType), stats.fontTypes),
```

Comment:
Would it make sense to do the swap *once* for each type, and cache that, rather than repeating it for every single page that's rendered?

Reply:
Yeah, I could do that. I've also considered just changing the constant int values to be strings, which would simplify all of this. This would technically be a major api change since we exposed these stats, but seeing that we didn't expose the StreamType and FontType constants, I doubt anyone is using them.

Reply:
> I've also considered just changing the constant int values to be strings, which would simplify all of this.

That does sound like the easiest solution here.

> This would technically be a major api change since we exposed these stats, but seeing that we didn't expose the StreamType and FontType constants, I doubt anyone is using them.

I wouldn't worry about that either, since I also don't believe that `getStats` is used much (or even at all) outside of mozilla-central.